src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -247,6 +247,74 @@ void HexagonGame::initLua_AudioControl()
         .doc(
             "Dives into the `Sounds` folder of the current level pack and "
             "plays the specified file `$0`.");
+
+    addLuaFn("a_syncMusicToDM", //
+        [this](bool value) {
+            levelStatus.syncMusicToDM = value;
+
+            auto* current(assets.getMusicPlayer().getCurrent());
+            setMusicPitch(current);
```

Comment:
Not sure if this can happen, but let's protect against undefined behavior:

```suggestion
            sf::Music* current(assets.getMusicPlayer().getCurrent());
            if (current == nullptr)
            {
                return:
            }
            
            setMusicPitch(*current);
```